docs/sphinx/dev_guide/ci.rst

```diff
@@ -8,13 +8,76 @@
 
 .. _ci:
 
-**********************
-Continuous Integration
-**********************
+************************************
+Continuous Integration (CI) Testing
+************************************
+
+The RAJA project employes multiple tools to run its tests for each GitHub
+*pull request*, all of which must pass before the pull request can be merged.
+These tools include:
+
+  * **Travis CI.** This runs builds for a Linux system for multiple versions of
+    the GNU and clang compilers, and one version of the Intel, nvcc (CUDA), and 
+    HIP compilers. RAJA tests are run each non-GPU build.
+  * **Appveyor.** This runs builds and tests for a Windows environment for two
+    versions of the Visual Studio compiler.
+  * **Gitlab CI.** Runs builds and tests on platforms in the Livermore
+    Computing *Collaboration Zone*. This is a recent addition for RAJA and
+    is a work-in-progress to get full coverage of compilers and tests we
+    need to exercise.
+
+These tools integrate seamlessly with GitHub. So they will automatically
+(re)run RAJA builds and tests as changes are pushed to each PR branch. Gitlab
+CI execution on Livermore Computing resources has some restrictions which are
+described below.
+
+Gitlab CI support is still being developed to make it more easy to use with 
+GitHub projects. The current state is described below.
+
+.. note:: The status of checks (pass/fail, running status) for each of these 
+          tools can be viewed by clicking the appropriate link in the check
+          section of a pull request.
 
 Gitlab CI
 =========
 
-RAJA shares its Gitlab CI workflow with other projects. The documentation is
-therefore `shared <https://radiuss-ci.readthedocs.io/en/latest/uberenv.html#ci>`_.
+If all memmbers of a GitHub project are members of the LLNL GitHub organization 
+and have tw-factor authentication enabled on their GitHub accounts, 
+auto-mirroring on the Livermore Computing Collaboration Zone Gitlab server is
+anabled. Thus, Gitlab CI will run automatically for those projects on pull 
+requested that are made by project members. Otherwise, due to Livermore 
+Computing security policies, Gitlab CI must be launched manually by a *blessed* 
+GitHub user satisfying the constraints described above. To manually initiate
+Gitlab CI on a pull request, add a comment with 'LGTM' in it.
+
+It is important to note that RAJA shares its Gitlab CI workflow with 
+other projects. See `Shared Gitlab CI Workflow <https://radiuss-ci.readthedocs.io/en/latest/uberenv.html#ci>`_ for more information.
+
+Vetted Specs
+------------
+
+The *vetted* compiler specs are those which we use during the RAJA Gitlab CI
+testing process. These can be viewed by looking at files in the RAJA
+``.gitlab`` directory. For example,
+
+.. code-block:: bash
+
+  $ ls -c1 .gitlab/*jobs.yml
+  .gitlab/lassen-jobs.yml
+  .gitlab/quartz-jobs.yml
+
+lists the yaml files containing the Gitlab CI jobs for lassen and quartz.
```

Comment:
We should add a note somewhere that soon this will be changed from quartz to ruby. Also, when I finally get the hip jobs working, corona will also be added.

Reply:
We can update the docs as we make changes.

Reply:
Ok, yea that'd be good. I'll add a note to my todo list for switching to ruby to also update docs